controllers/common/ansible.go

```diff
@@ -5,26 +5,111 @@ package common
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"reflect"
 	"regexp"
+	"sort"
+	"strconv"
 	"strings"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
+	ctrl "sigs.k8s.io/controller-runtime"
 
 	policyv1beta1 "open-cluster-management.io/governance-policy-propagator/api/v1beta1"
 )
 
+const (
+	ControllerName             string = "policy-automation"
+	PolicyAutomationLabel      string = "ansiblejob.policy.open-cluster-management.io/policyautomation-name"
```

Comment:
I'd think this would be PolicyAutomation instead of AnsibleJob?
```suggestion
	PolicyAutomationLabel      string = "policyautomation.policy.open-cluster-management.io/policyautomation-name"
```

Reply:
This label attaches to ansiblejob not policyautomation. In other words, add "policyautomation name" on ansiblejob

Reply:
good good let me double check

Reply:
(Similar to the other keys here)

Reply:
I changed to "policiy" :)